### ISSUE TITLE
Failed message doesn't render the @mention inline avatar

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/spreed#7973]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Composer a chat message including a \@mention of another user
2. See the user avatar
3. Send the message
4. ??? server error ???
5. See that the message wasn't sent with the option to resend

### Expected behaviour

Message to resend renders the \@mention

### Actual behaviour

I see \@joas in the chat window

## Talk app

**Talk app version:** the lastest for 24 I guess

**Custom Signaling server configured:** yes/no and version (see additional admin settings: `/index.php/index.php/settings/admin/talk#signaling_server`)

**Custom TURN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#turn_server`)

**Custom STUN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#stun_server`)

## Browser

**Microphone available:** yes/no

**Camera available:** yes/no

**Operating system:** Windows/Ubuntu/...

**Browser name:** Firefox/Chrome/...

**Browser version:** 85/96/...

### Browser log

<details>
```
Insert your browser log here, this could for example include:
a) The javascript console log
b) The network log
c) ...
```

</details>

## Server configuration

**Operating system**: Ubuntu/RedHat/...

**Web server:** Apache/Nginx

**Database:** MySQL/Maria/SQLite/PostgreSQL

**PHP version:** 7.4/8.0/8.1

**Nextcloud Version:** (see admin page)

**List of activated apps:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your server installation folder
```
</details>

**Nextcloud configuration:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
```
</details>

### Server log (data/nextcloud.log)
<details>

```
Insert your server log here
```
</details>

**Comment by nickvergessen** [comment]
Yeah, so the problem is the message never hit the server so we also don't know which parameters or magic words are recognized etc. If we would replace them and keep a record of the rich-object parameters, that would break resending the message if not re-replaced on re-sending.

**Comment by nickvergessen** [comment]
Might be related to https://github.com/nextcloud/spreed/issues/10895

**Comment by Antreesy** [comment]
> the message never hit the server

As you said, we don't have messageParameters yet to show a message, although that could be mocked by client.

> we could only do it for mentions you autocompleted via tribute

Only if we imitate somehow the `click` on a tribute to generate a mention chip. But that would require a server request for autocomplete options.
Issue becomes more relevant now, as we added message editing

**Comment by Antreesy** [comment]
Oh, it's actually tracked in #2714, as both are temporary messages
